docs/upgrade.md

```diff
@@ -1,3 +1,44 @@
+# 5.x Upgrade Notes
```

Comment:
6.x right ?

Reply:
No I meen 5.x as it is a deprecation in 5.x and adding a alias for ```connect()``` method, as the commit is targeting 5.x branch.

I need to make PR for the doc changes in 6.x which explain the new connect method.